Extensions/PathfindingBehavior/tests/legacypathfindingruntimebehavior.spec.js

```diff
@@ -0,0 +1,689 @@
+    variables: [],
+    behaviorsSharedData: [],
+    objects: [],
+    instances: [],
+  });
+  runtimeScene._timeManager.getElapsedTime = function () {
+    return (1 / 60) * 1000;
+  };
+
+  const pathFindingName = 'auto1';
+  const player = new gdjs.SpriteRuntimeObject(runtimeScene, {
+    name: 'player',
+    type: '',
+    animations: [
+      {
+        name: 'animation',
+        directions: [
+          {
+            sprites: [
+              {
+                originPoint: objectCenteredOnCells
+                  ? { x: 80, y: 80 }
+                  : { x: 87, y: 87 },
+                centerPoint: { x: 80, y: 80 },
+                points: [
+                  { name: 'Center', x: 80, y: 80 },
+                  objectCenteredOnCells
+                    ? { name: 'Origin', x: 80, y: 80 }
+                    : { name: 'Origin', x: 87, y: 87 },
+                ],
+                hasCustomCollisionMask: false,
+              },
+            ],
+          },
+        ],
+      },
+    ],
+    behaviors: [
+      {
+        type: 'PathfindingBehavior::PathfindingBehavior',
+        name: pathFindingName,
+        allowDiagonals: false,
+        acceleration: 400,
+        maxSpeed: 200,
+        angularMaxSpeed: 180,
+        rotateObject: false,
+        angleOffset: 0,
+        cellWidth: cellSize,
+        cellHeight: cellSize,
+        extraBorder: 0
+      },
+    ],
+  });
+  player.getWidth = function () {
+    return 160;
+  };
+  player.getHeight = function () {
+    return 160;
+  };
+  runtimeScene.addObject(player);
+
+  const obstacle = addObstacleSprite3(runtimeScene, objectCenteredOnCells);
+
+  let playerLeftBorder;
+  let playerRightBorder;
+  let playerTopBorder;
+  let playerBottomBorder;
+
+  let obstacleLeftBorder;
+  let obstacleRightBorder;
+  let obstacleTopBorder;
+  let obstacleBottomBorder;
+
+  if (cellSize == 160) {
+    if (objectCenteredOnCells) {
+      playerLeftBorder = 80;
+      playerRightBorder = 80;
+      playerTopBorder = 80;
+      playerBottomBorder = 80;
+    } else {
+      playerLeftBorder = 87;
+      playerRightBorder = 73;
+      playerTopBorder = 87;
+      playerBottomBorder = 73;
+    }
+  } else {
+    // The player radius is at most 2 cells
+    playerLeftBorder = 2 * cellSize;
+    playerRightBorder = 2 * cellSize;
+    playerTopBorder = 2 * cellSize;
+    playerBottomBorder = 2 * cellSize;
+  }
+
+  if (objectCenteredOnCells) {
+    obstacleLeftBorder = 80;
+    obstacleRightBorder = 80;
+    obstacleTopBorder = 80;
+    obstacleBottomBorder = 80;
+  } else {
+    obstacleLeftBorder = 87;
+    obstacleRightBorder = 73;
+    obstacleTopBorder = 87;
+    obstacleBottomBorder = 73;
+  }
+
+  let inclusiveMargin;
+  if (cellSize == 160) {
+    // The collision is strict
+    inclusiveMargin = 0;
+  } else {
+    // The collision is not strict
+    inclusiveMargin = 1;
+  }
+
+  // multiple of 20 and 160
+  const playerX = 480;
+  const playerY = 320;
+
+  // This one is not on limit
```

Comment:
What do you mean exactly by "not on limit"?

Reply:
This test is not part of a test couple : there it works, there it doesn't. But it seems like an interesting case nonetheless. Though, I'm not sure.

Reply:
It's surely a good idea to keep it yes! Let's just remove the comment, as it's a test like the others.